pkg/kubelet/pluginmanager/pluginwatcher/plugin_watcher.go

```diff
@@ -93,14 +88,6 @@ func (w *Watcher) Start(stopCh <-chan struct{}) error {
 				}
 				continue
 			case <-stopCh:
-				// In case of plugin watcher being stopped by plugin manager, stop
-				// probing the creation/deletion of plugin sockets.
-				// Also give all pending go routines a chance to complete
-				select {
-				case <-w.stopped:
```

Comment:
`w.stopped` would only be closed in line # 76, but it won't be closed as # 76 is defer-ed. 

I am not sure what the original purpose of `w.stopped` is, but it seems to be useless now.

Reply:
I agree the stopped channel use was weird, so the `<-w.stopped` case would never have fired. That means the 11 second timeout would always have been hit.

This PR is removing that timeout, meaning the fsWatcher is closing immediately with no timeout. Is that correct? Does that cut off FS events we wanted to see/handle?

Reply:
I am not quite confident, but since the `stopCh` is closed, I think the plugin manager which starts the watcher expects the watcher to exit regardless of any further events.

Reply:
Yeah, I think you're right. Would like an ack from a sig-node reviewer

Reply:
yes, that is what we want.